.nuget/NuGet.targets

```diff
@@ -12,8 +12,6 @@
         <!-- Determines if package restore consent is required to restore packages -->
         <RequireRestoreConsent Condition=" '$(RequireRestoreConsent)' != 'false' ">false</RequireRestoreConsent>
 
-        <!-- Download NuGet.exe if it does not already exist -->
```

Comment:
CI was failing during nuget install because it was an older version. I've changed this to just use the local nuget.exe.